src/content/docs/en/recipes/getImage.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Build a custom Picture Component 
+description: Learn how to build a custom Picture Component using the getImage function
+i18nReady: true
+type: recipe
+---
+
+The `getImage()` function is used for generating images that will be used elsewhere instead of directly in the HTML. In this recipe, you will learn how to create your own `<Picture>` component using the `getImage()` function.
+
+
+## Prerequisites
+- A project with [SSR](/en/guides/server-side-rendering/) (`output: 'server'`) enabled
```

Comment:
I don't see the link between SSR and this recipe, image optimization works in SSG.